CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+## Head
+
+- **[Breaking change]**: Add `skipFragment` option, defaulting to `false`.
```

Comment:
We need to change this text to "defaulting to `true`"

Reply:
Now you understand that I was very turned around here. :|

Reply:
No problem!